src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/IcraEligibility.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-container>
+    <div>
+      <Breadcrumb />
+      <h1 class="pb-9">Apply with international certification</h1>
+
+      <ECEHeader title="Is this pathway right for you?" />
+      <p>
+        If you are certified as an Early Childhood Educator (ECE) in another country, you may be eligible to apply for ECE Five Year certification in British
+        Columbia using professional experience you have outside of Canada.
+      </p>
+      <p class="mt-5 mb-8">
+        <b>Note:</b>
+        This application is only for
+        <b>ECE Five Year certification.</b>
+        It does not apply to ECE Assistant of ECE One Year certification.
+      </p>
+      <ECEHeader title="The eligibility process" />
```

Comment:
for the numbered list like 
1.
2.
3. 

I would suggest using ordered list. The number will be automatically generated for you. 
```
<ol class="ml-10">
<li/>
<li />
</ol>
```